packages/cli-platform-apple/src/commands/runCommand/getBuildPath.ts

```diff
@@ -1,38 +1,13 @@
-import child_process from 'child_process';
-import {IOSProjectInfo} from '@react-native-community/cli-types';
-import {CLIError, logger} from '@react-native-community/cli-tools';
-import chalk from 'chalk';
+import {CLIError} from '@react-native-community/cli-tools';
+import path from 'path';
 
 export async function getBuildPath(
-  xcodeProject: IOSProjectInfo,
-  mode: string,
-  buildOutput: string,
-  scheme: string,
-  target: string | undefined,
+  buildSettings: any,
```

Comment:
can we refine that type?

Reply:
Shouldn't this be of type `BuildSettings` from [here](https://github.com/react-native-community/cli/pull/2234/files#diff-48db413602ba59c3e5026b1d3cb34d52c390469d652b9b0c20f7e2f0a538bfb2R6)